project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "build":"docker-compose up -d postgis",
```

Comment:
I wouldn't put this in a build command b/c it's not related to building the project.  Seems like you want a `pre-run` command, or add it to the beginning of the existing `run` command.  Also, the build command for the server would be `lein uberjar` and the command to build the web ui/dashboard would be `npm run build:dev`, for the dev environment